src/components/radio/Radio.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, createEffect, createSignal } from 'solid-js'
+import type { JSX } from 'solid-js'
+
+import StyledIcon from './StyledIcon'
+import StyledInput from './StyledInput'
+import StyledLabel from './StyledRadio'
+
+export type RadioProps = {
+  label?: JSX.Element
+  labelPosition?: 'right' | 'left'
+  checked?: boolean
+  indeterminate?: boolean
+  disabled?: boolean
+  className?: string
```

Comment:
Is this className prop being used anywhere?

Reply:
No, I've removed it.